lisp/plugins/action_cues/collection_cue.py

```diff
@@ -82,12 +111,20 @@ def __init__(self, **kwargs):
         self.collectionView.setAlternatingRowColors(True)
         self.collectionGroup.layout().addWidget(self.collectionView)
 
+        self.bottomBarGroup = QWidget(self)
+        self.bottomBarGroup.setLayout(QHBoxLayout())
```

Comment:
```suggestion
        self.bottomBarGroup.setLayout(QHBoxLayout())
        self.bottomBarGroup.layout().setContentsMargins(0, 0, 0, 0)
```

We can remove the margins of the inner layout